Automattic/vip-cli
# Fix interactive commands not working correctly

## Description

This fixes problems running commands that require user input (such as `wp user create --prompt`). 

## Steps to Test

1. Check out PR.
2. Run `npm run build`
3. Run `/dist/bin/vip-wp.js --yes  @site.env --  user create --prompt`
4. Confirm that entering data at a prompt results in the next prompt (or command completion)

Fixes #464
